docs/public/presets/toml/nerd-font-symbols.toml

```
@@ -171,6 +171,9 @@ symbol = "󱘗 "
 [scala]
 symbol = " "
 
+[status]
+symbol = " "
```

The default emoji does not have a space:
```suggestion
symbol = ""
```